src/extension/options-page/NewPersonaCard.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { makeStyles, createStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+import { Link } from 'react-router-dom'
+import CardContent from '@material-ui/core/CardContent'
+import AddIcon from '@material-ui/icons/Add'
+import Button from '@material-ui/core/Button'
+import { Avatar } from '@material-ui/core'
+const useStyles = makeStyles(theme =>
+    createStyles({
+        card: {
+            minWidth: 375,
```

Comment:
on mobile?

Reply:
> on mobile?

iPhone 8 is 375. Should be small enough for 2019.